src/core/or/sendme.c

```diff
@@ -286,6 +286,29 @@ send_circuit_level_sendme(circuit_t *circ, crypt_path_t *layer_hint,
   return 0;
 }
 
+/*
+ * Public API
+ */
+
+/** Return true iff the next cell for the given cell window is expected to be
+ * a SENDME.
+ *
+ * We are able to know that because the package or deliver window value minus
+ * one cell (the possible SENDME cell) should be a multiple of the increment
+ * window value. */
+bool
+sendme_circuit_is_next_cell(int window)
```

Comment:
Maybe next_cell_is_sendme would be a good name?

Reply:
I want to namespace it ... so it would end up a bit confusing like `sendme_next_cell_is_sendme()`. Also, the `circuit` in there I would like to keep because it cannot be used for a stream level SENDME check.

What about `sendme_circuit_cell_is_next()` ?

If you are ok with it, see fixup commit: 17636edb33]